<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_Monitor</class>
 <widget class="QWidget" name="Frm_Monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font: 14pt;
font-family: &quot;微软雅黑&quot;;
}

QWidget {
color: #c2c2c3;
background: #343639;
}

QTabWidget::tab-bar {
	left: 15px;
}

QTabWidget::pane {
border: 1px solid black;
}

QTabBar::tab{
background: #343639;
border: 1px solid black;
border-bottom-color: #343639; 
border-top-left-radius: 4px;
border-top-right-radius: 4px;
min-width: 70px;
padding: 2px;
min-height: 15px;
}

QTabBar::tab:selected {
color: white;
padding-top: 4px;
background: #2773b4;
border-bottom: #2773b4;
}

QGroupBox {
border: 1px solid black;
border-radius: 1px
}

QGroupBox::title {
   
}


QCheckBox {
outline: no;
     spacing: 5px;
}

QCheckBox::indicator {
     width: 30px;
     height: 30px;
}

#tab_input QCheckBox::indicator:unchecked {
     image: url(:/Res/image/btn_cb_io_30x30_off.png);
}

#tab_input QCheckBox::indicator:checked {
     image: url(:/Res/image/btn_cb_io_30x30_on.png);
}

#tab_output QCheckBox::indicator:unchecked {
     image: url(:/Res/image/btn_cb_io_30x30_off.png);
}

#tab_output QCheckBox::indicator:checked {
     image: url(:/Res/image/btn_cb_io_30x30_on.png);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_input">
      <attribute name="title">
       <string>输入</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbI1">
         <property name="text">
          <string>输入1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="cbI10">
         <property name="text">
          <string>输入10</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="cbI19">
         <property name="text">
          <string>输入19</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="cbI28">
         <property name="text">
          <string>输入28</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="cbI37">
         <property name="text">
          <string>输入37</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cbI2">
         <property name="text">
          <string>输入2</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbI11">
         <property name="text">
          <string>输入11</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="cbI20">
         <property name="text">
          <string>输入20</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="cbI29">
         <property name="text">
          <string>输入29</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QCheckBox" name="cbI38">
         <property name="text">
          <string>输入38</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="cbI3">
         <property name="text">
          <string>输入3</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="cbI12">
         <property name="text">
          <string>输入12</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="cbI21">
         <property name="text">
          <string>输入21</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QCheckBox" name="cbI30">
         <property name="text">
          <string>输入30</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QCheckBox" name="cbI39">
         <property name="text">
          <string>输入39</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="cbI4">
         <property name="text">
          <string>输入4</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="cbI13">
         <property name="text">
          <string>输入13</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="cbI22">
         <property name="text">
          <string>输入22</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QCheckBox" name="cbI31">
         <property name="text">
          <string>输入31</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QCheckBox" name="cbI40">
         <property name="text">
          <string>输入40</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="cbI5">
         <property name="text">
          <string>输入5</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="cbI14">
         <property name="text">
          <string>输入14</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="cbI23">
         <property name="text">
          <string>输入23</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QCheckBox" name="cbI32">
         <property name="text">
          <string>输入32</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QCheckBox" name="cbI41">
         <property name="text">
          <string>输入41</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="cbI6">
         <property name="text">
          <string>输入6</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="cbI15">
         <property name="text">
          <string>输入15</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="cbI24">
         <property name="text">
          <string>输入24</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QCheckBox" name="cbI33">
         <property name="text">
          <string>输入33</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QCheckBox" name="cbI42">
         <property name="text">
          <string>输入42</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="cbI7">
         <property name="text">
          <string>输入7</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="cbI16">
         <property name="text">
          <string>输入16</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="cbI25">
         <property name="text">
          <string>输入25</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QCheckBox" name="cbI34">
         <property name="text">
          <string>输入34</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QCheckBox" name="cbI43">
         <property name="text">
          <string>输入43</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="cbI8">
         <property name="text">
          <string>输入8</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="cbI17">
         <property name="text">
          <string>输入17</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="cbI26">
         <property name="text">
          <string>输入26</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QCheckBox" name="cbI35">
         <property name="text">
          <string>输入35</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QCheckBox" name="cbI44">
         <property name="text">
          <string>输入44</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="cbI9">
         <property name="text">
          <string>输入9</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="cbI18">
         <property name="text">
          <string>输入18</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QCheckBox" name="cbI27">
         <property name="text">
          <string>输入27</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QCheckBox" name="cbI36">
         <property name="text">
          <string>输入36</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_output">
      <attribute name="title">
       <string>输出</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="rightMargin">
        <number>160</number>
       </property>
       <property name="horizontalSpacing">
        <number>9</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="cbO1">
         <property name="text">
          <string>输出1</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="cbO10">
         <property name="text">
          <string>输出10</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="cbO19">
         <property name="text">
          <string>输出19</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="cbO28">
         <property name="text">
          <string>输出28</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="cbO2">
         <property name="text">
          <string>输出2</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbO11">
         <property name="text">
          <string>输出11</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="cbO20">
         <property name="text">
          <string>输出20</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="cbO3">
         <property name="text">
          <string>输出3</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="cbO12">
         <property name="text">
          <string>输出12</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="cbO21">
         <property name="text">
          <string>输出21</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="cbO4">
         <property name="text">
          <string>输出4</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="cbO13">
         <property name="text">
          <string>输出13</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="cbO22">
         <property name="text">
          <string>输出22</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="cbO5">
         <property name="text">
          <string>输出5</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="cbO14">
         <property name="text">
          <string>输出14</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="cbO23">
         <property name="text">
          <string>输出23</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="cbO6">
         <property name="text">
          <string>输出6</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="cbO15">
         <property name="text">
          <string>输出15</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="cbO24">
         <property name="text">
          <string>输出24</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="cbO7">
         <property name="text">
          <string>输出7</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="cbO16">
         <property name="text">
          <string>输出16</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="cbO25">
         <property name="text">
          <string>输出25</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="cbO8">
         <property name="text">
          <string>输出8</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QCheckBox" name="cbO17">
         <property name="text">
          <string>输出17</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="cbO26">
         <property name="text">
          <string>输出26</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="cbO9">
         <property name="text">
          <string>输出9</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QCheckBox" name="cbO18">
         <property name="text">
          <string>输出18</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QCheckBox" name="cbO27">
         <property name="text">
          <string>输出27</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
